pkg/apis/config/defaults.go

```diff
@@ -137,6 +140,8 @@ type Defaults struct {
 
 	UserContainerNameTemplate string
 
+	ContainerConcurrency int64
```

Comment:
See Matt's comment here: https://github.com/knative/serving/pull/4911#discussion_r308009511

You'll need to add logic to NewDefaultsConfigFromMap to handle ContainerConcurrency and docs/examples to config/config-defaults.yaml